device.c

```diff
@@ -80,7 +83,7 @@ int adev_open(const hw_module_t* module, const char* name, hw_device_t** device)
 
     if (ref_counter != 0) {
         *device = &xadev->hw_device.common;
-        ALOGD("Return existing device:%p", *device);
+        LOG_FN_PARAMETERS("Return existing device:%p", *device);
```

Comment:
Don't think this and below changes of ALOG->LOG_FN are relevant

Reply:
This change is removed from pull request, and will be submitted separately.